packages/failurechecker/src/failurechecker.ts

```diff
@@ -37,43 +36,54 @@ const START_HOUR_UTC = 17;
 export = (app: Application) => {
   app.on(['schedule.repository'], async context => {
     const utcHour = moment.utc().hour();
+    const owner = context.payload.organization.login;
+    const repo = context.payload.repository.name;
 
-    if (utcHour > END_HOUR_UTC && utcHour < START_HOUR_UTC) {
+    // If we're outside of working hours, and we're not in a test context, skip this bot.
+    if (
+      utcHour > END_HOUR_UTC &&
+      utcHour < START_HOUR_UTC &&
+      process.env.NODE_ENV !== 'test'
```

Comment:
otherwise tests only work during working hours :laughing:

Reply:
I'd rather mock `Date`, but this is fine too :)